aws-lc-fips-sys/builder/cmake_builder.rs

```diff
@@ -0,0 +1,196 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+use crate::OutputLib::{Crypto, RustWrapper, Ssl};
+use crate::{target, target_arch, target_os, target_vendor, test_command, OutputLibType};
+use std::env;
+use std::ffi::OsStr;
+use std::path::PathBuf;
+
+pub(crate) struct CmakeBuilder {
+    manifest_dir: PathBuf,
+    out_dir: PathBuf,
+    build_prefix: Option<String>,
+    output_lib_type: OutputLibType,
+}
+
+fn test_perl_command() -> bool {
+    test_command("perl".as_ref(), &["--version".as_ref()])
+}
+
+fn test_go_command() -> bool {
+    test_command("go".as_ref(), &["version".as_ref()])
```

Comment:
do we need to enforce minimum versions here?

i suppose similar question applies to non-fips